Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -177,18 +177,14 @@ extension NewsTableViewController: UITableViewDataSource {
 
         if let imgURL = viewModel.imageURL {
             cell.newsImageView.isHidden = false
-            cell.newsImageView.sd_setImage(
-                with: imgURL,
-                placeholderImage: UIImage(named: "icon_placeholder_small"),
-                options: [],
-                completed: { (_, error, _, _) in
-
-                    if error != nil {
-                        return
-                    }
-
+            cell.newsImageView.kf.setImage(with: imgURL, placeholder: UIImage(named: "icon_placeholder_small")) { result in
+                switch result {
+                case .success(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/Controllers/NewsTableViewController.swift#L182"></span>Arguments can be omitted when matching enums with associated types if they are not used.<br />
<code>empty_enum_arguments</code> <code>NewsTableViewController.swift:182</code></td>
    </tr>
  </tbody>
</table>